src/Exception/ExceptionFactory.php

```diff
@@ -78,6 +96,6 @@ private static function generateErrorMessage(int $responseCode, array $messages)
      */
     private static function glueMessages(array $messages): string
     {
-        return implode(self::GLUE_ERROR_MESSAGES, $messages);
+        return implode(self::SEPARATOR_ERROR_MESSAGES, $messages);
```

Comment:
Would call it `ERROR_MESSAGE_SEPARATOR` then, (big -> small)

Reply:
We normally group constants to getter, so in this case this constant belongs to the `SEPARATOR_` constant group. As you can have multiple separators and then the constant names will begin with `SEPARATOR_XXXXXX`.